static/js/util/validation.js

```diff
@@ -277,8 +288,8 @@ export function validateYear(string: string): ?number {
 /**
  * Returns a function which merges the results of the given functions on a set of arguments
  */
-export function combineValidators(...validators: Array<UIValidator|Validator>): UIValidator|Validator {
-  return (...args: Array<Profile|UIState>) => _.merge({}, ...validators.map(
+export function combineValidators(...validators: Array<Function>): Function {
+  return (...args) => _.merge({}, ...validators.map(
     validator => validator(...args)
   ));
 }
```

Comment:
Unfortunately I haven't been able to get any more specific annotation to type check for this function ; _ ;